edx-platform/teclisboa/lms/static/sass/partials/lms/theme/_extras.scss

```diff
@@ -378,6 +378,19 @@
 	}
 }
 
+// Hide link/unlink to/from `Técnico ID` in account page
+div.u-field.u-field-social.u-field-auth-saml-istx-sso {
```

Comment:
Como es el mismo estilo para todas lo que puedes hacer es poner una coma finalizando y luego agregar los otros selectores css: https://www.w3schools.in/css3/group-selector/

Reply:
Claro! Muchas gracias.